app/controllers/presentations_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.page(params[:page]).order(:created_at)
+    @presentations = Presentation.page(params[:page]).order(created_at: :desc)
```

Comment:
Should it use the `created_at` or the actual date of the presentation? I'd prefer to see them in order of when they were actually presented. What do you think?